src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser.md

```diff
@@ -86,9 +86,17 @@ $ k6 run script.js
 ```
 
 ```bash
-# When using the `k6:master-with-browser` Docker image, you need to add `--cap-add=SYS_ADMIN`
-# to grant further system permissions on the host for the Docker container.
-docker run --rm -i --cap-add=SYS_ADMIN grafana/k6:master-with-browser run - <script.js
+# WARNING!
+# Be aware that grafana/k6:master-with-browser launches Chrome browser by setting the
+# 'no-sandbox' argument. Use it only with trustworthy websites.
+#
+# As an alternative, our recommendation is to use a Docker SECCOMP profile instead, and
```

Comment:
```suggestion
# As an alternative, our recommendation is to use a Docker SECCOMP profile instead and
```

Reply:
I applied the suggestion mentioned in https://github.com/grafana/k6-docs/pull/1330#discussion_r1332210819. See [7b98b68](https://github.com/grafana/k6-docs/pull/1330/commits/7b98b68942a04393fb1e2f587c2e1f0d570d61e3).